modules/administration-guide/pages/configuring-routes.adoc

```diff
@@ -6,20 +6,16 @@
 [id="configuring-routes_{context}"]
 = Configuring OpenShift Route to work with Router Sharding
 
-This procedure describes how to configure labels, annotations, and domains for OpenShift Route to work with link:https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#nw-ingress-sharding_configuring-ingress[Router Sharding]. The chapter then mentions the configuration process for existing instances or those about to be installed.
+This procedure describes how to configure labels, annotations, and domains for OpenShift Route to work with link:https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#nw-ingress-sharding_configuring-ingress[Router Sharding].
```

Comment:
```suggestion
You can configure labels, annotations, and domains for OpenShift Route to work with link:https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#nw-ingress-sharding_configuring-ingress[Router Sharding].
```
IBM SG: "Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials."